### PR TITLE
Added example for lod_to_tensor and tensor_to_lod

### LINKED ISSUE #723: LoD-Tensor doc python version not uniform
In the file below, the python version is not uniform.
https://github.com/PaddlePaddle/FluidDoc/blob/develop/doc/fluid/user_guides/howto/basic_concept/lod_tensor.rst

python2 style:
image

python3 style:
image

### PR DESCRIPTION
**Pull request by tink2123**
* add an example for tensor to lod_tensor
* add an example for lod_tensor to tensor
* fix #723 

预览效果：
image

### CODE REVIEW

File: doc/fluid/user_guides/howto/basic_concept/lod_tensor.rst [hunk 1]
```diff
@@ -153,6 +153,10 @@ LoD-Tensor
 
 recursive_seq_lens 是一个双层嵌套列表，也就是列表的列表，最外层列表的size表示嵌套的层数，也就是lod-level的大小；内部的每个列表，对应表示每个lod-level下，每个元素的大小。
 
+下面三段代码分别介绍如何创建一个LoD-Tensor、如何将LoD-Tensor转换成Tensor、如何将Tensor转换成LoD-Tensor：
```
Comment (heavengate) [inline doc/fluid/user_guides/howto/basic_concept/lod_tensor.rst]: 、-> ，

**Review by heavengate — COMMENTED** [review]
pr里面需要贴上文档预览

File: doc/fluid/user_guides/howto/basic_concept/lod_tensor.rst (hunk 1 above)
Comment (tink2123) [reply doc/fluid/user_guides/howto/basic_concept/lod_tensor.rst]: done

**Review by heavengate — APPROVED** [review]
LGTM

**Comment by tink2123** [comment]
@shanyi15 pelease review, thanks

**Review by shanyi15 — APPROVED** [review]
LG

**Pull request merged**